index.html

```diff
@@ -221,6 +221,50 @@ <h2>About me</h2>
             </ul>
         </div>
     </section>
+    <section class="portfolio-form">
+        <div class="form-text">
+            <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+        </div>
+        <div class="form-area">
+            <form action="https://formspree.io/f/xzboreoy" method="post">
+                <ul class="contact-form">
+                    <li class="name-input">
+                        <input class="names" type="text" name="First Name" maxlength="30" placeholder="First Name">
+                        <input class="names" type="text" name="Last Name" maxlength="30" placeholder="Second Name">
+                    </li>
+                    <li class="full-name-input">
+                        <input class="full-name" type="text" name="name" placeholder="Full name">
+                    </li>
+                    <li>
+                        <input class="email" type="email" name="Email address" placeholder="Email address">
```

Comment:
- [ ] All input fields must have the `required `attribute to enable validation on submission. let's go, please kindly add them🧑‍🔧
- Example : 
```
<input class="email" type="email" name="Email address" placeholder="Email address" required >
```